### PR TITLE
update browseClass and add myclass

### PR DESCRIPTION
**Pull request by avan989**

### CODE REVIEW

File: T2CC_APP/t2cc/app/src/main/res/layout/activity_myclass.xml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        app:layout_constraintStart_toEndOf="@+id/myClassArrow"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.0" />
+
+    <TextView
+        android:id="@+id/myClassBodyLabel"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="28dp"
+        android:text="Active/Subscribed Classes "
```
Comment (olabusayo) [inline T2CC_APP/t2cc/app/src/main/res/layout/activity_myclass.xml:52]: I think we should pick one of the other. I'm partial to Subscribed

File: T2CC_APP/t2cc/app/src/main/res/layout/activity_browseclass.xml [hunk 2]
```diff
@@ -50,75 +50,72 @@
...
             android:textStyle="bold" />
 
         <TextView
             android:id="@+id/subUnsubLabel"
-            android:layout_width="171dp"
+            android:layout_width="80dp"
             android:layout_height="wrap_content"
             android:layout_weight="0"
-            android:text="Subscribe/Unsubscribe "
+            android:text="Toggle To Subscribe"
```
Comment (olabusayo) [inline T2CC_APP/t2cc/app/src/main/res/layout/activity_browseclass.xml:97]: I think just "Subscribe" would be fine here

File: T2CC_APP/t2cc/app/src/main/res/layout/activity_browseclass.xml [hunk 3]
```diff
@@ -50,75 +50,72 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="17dp"
-        android:layout_marginBottom="46dp"
         android:text="Classes Available For Subscription: "
```
Comment (olabusayo) [inline T2CC_APP/t2cc/app/src/main/res/layout/activity_browseclass.xml:53]: I think just "Available Classes" would do.

**Comment by avan989** [comment]
updated

**Pull request merged**